.packit.yaml

```diff
@@ -27,6 +31,13 @@ packages:
     specfile_path: python-copr-common.spec
     files_to_sync:
       - python-copr-common.spec
+    actions:
+      create-archive:
+        - bash -c "tito build --tgz --test -o ."
+        - bash -c "ls -1t ./*.tar.gz | head -n 1"
+      get-current-version:
+        - bash -c "git describe --match python-copr-common-[0-9]* --abbrev=0 HEAD | egrep -o
+          [0-9]+\.[0-9]+"
```

Comment:
We could happily use this at global level if we could specify `--match` a generic way.  Can't we use something like `{{package_name}}-[0-9]*`?

Reply:
currently it is not possible :/ (looks like related to https://github.com/packit/packit/issues/1925) however, do you need to introduce an action for this? The default way Packit gets the version when no action is specified is by extracting it from the tag, and these can be filtered using `upstream_tag_include` (that you have already defined)